src/parser.ts

```diff
@@ -440,3 +372,31 @@ function getParamsOrders(ast: AST, tokenIndex: number) {
 
   return paramsOrder;
 }
+
+/**
+ * This converts the `name`, `optional`, and `default` properties into a single `name` property.
+ */
+function combineName(tag: commentParser.Tag): void {
+  // Additional operations on name
+  if (tag.name) {
```

Comment:
Please destruct tag like `function combineName({tag,name,optional,default:_default}: commentParser.Tag)`

Reply:
The function changes the tag object itself. Destructuring is not possible.